evm/utils/numeric.py

```diff
@@ -21,6 +29,17 @@ def int_to_byte(value):
     return bytes([value])
 
 
+def int_to_bytes32(value):
+    value_bytes = pipe(
+        value,
+        int_to_big_endian,
+        pad32,
+    )
+    if len(value_bytes) > 32:
+        raise ValueError("value is too large")
```

Comment:
Instead of doing *post* validation, how about just adding a pre-condition check that it's a valid `uint256` and raising a `ValueError` if it isn't.  (hint, I don't think we should use `evm.validation.validate_uint256` here since it's outside of `evm.utils`.  Instead, just inline the same validation code.

Btw, I suggest `ValueError` for the same reason, not importing from outside of `evm.utils` from within `evm.utils`.

Reply:
Thank you for suggesting this.
Actually I'm curious how to know when to import one function from outside directory or just inline it?
I can guess in this case the validation is simple so the cost(DRY) is not expensive. But if it is in other case, is it not good to import function in another layer?